### PR TITLE
[v6r13] Fixes to DMSHelpers

### PR DESCRIPTION
**Pull request by phicharp**
The module Core.Utilities.SiteSEMapping was updated to make use of the DMSHelpers module. This should ease the porting of all components to using this single source of information.
DataManager was updated to use directly DMSHelpers

### CODE REVIEW

File: Resources/Storage/StorageFactory.py [hunk 1]
```diff
@@ -35,6 +35,12 @@ def __init__( self, useProxy = False, vo = None ):
         self.vo = result['Value']
       else:
         RuntimeError( "Can not get the current VO context" )
+    self.remotePlugins = []
```
Comment (chaen) [inline Resources/Storage/StorageFactory.py]: These are already initialized in getStorages. Can it be initialized in one place only ? The __init__ is probably the best one

File: Resources/Storage/StorageFactory.py [hunk 2]
```diff
@@ -93,7 +101,6 @@ def getStorages( self, storageName, pluginList = [] ):
     # In case the storage is made from a base SE, get this information
     res = self._getConfigStorageName( storageName, 'BaseSE' )
     if not res['OK']:
-      self.valid = False
```
Comment (chaen) [inline Resources/Storage/StorageFactory.py]: why is that removed ? Is this attribute even used anywhere ?

File: Resources/Storage/StorageFactory.py [hunk 3]
```diff
@@ -253,13 +260,13 @@ def _getConfigStorageProtocolDetails( self, storageName, protocolSection ):
...
-        
+
     # This is a temporary for backward compatibility
-    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:  
+    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:
       protocolDict['PluginName'] = protocolDict['ProtocolName']
-    protocolDict.pop( 'ProtocolName' )  
-        
-    # Evaluate the base path taking into account possible VO specific setting 
+    protocolDict.pop( 'ProtocolName' )
```
Comment (chaen) [inline Resources/Storage/StorageFactory.py]: Either this has to go inside the "if" statement, or in a try catch, or pop('ProtocolName', None)

File: Resources/Storage/StorageFactory.py (hunk 2 above)
Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: No, this is why I removed it ;-)

File: Resources/Storage/StorageFactory.py (hunk 1 above)
Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: Yes I know... I had no idea whether it was required to re-initialize them in getStorages. As it is it doesn't harm and makes the number of pylint warnings decrease... Someone who has time/expertise should check whether they can be removed from getStorages ;-)

File: Resources/Storage/StorageFactory.py (hunk 3 above)
Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: There is no change here on my side... Just Eclipse beautifying (although I have no clue why the comment disappeared ;-). I will check the logic in here...

Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: In my Eclipse the comment is not removed, just a few spaces removed in the 2 preceding lines... strange :(

Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: B.t.w. it would be good if everybody could avoid adding blank spaces here and there... overtime Eclipse is removing them and creates an artificial change which makes reviewing difficult... at least one good point with Eclipse ;-)

Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: Oups, sorry... just checked... Indeed the comment was there twice and I removed it once, so all is OK

File: Resources/Storage/StorageFactory.py (hunk 1 above)
Comment (chaen) [inline Resources/Storage/StorageFactory.py]: From a quick look, it looks like they don't need to be reinitialized.

Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: OK, I'll remove them after a second check...

Comment (phicharp) [inline Resources/Storage/StorageFactory.py]: Removed

**Comment by chaen** [comment]
review OK
